Makefile

```diff
@@ -47,7 +47,7 @@ $(BUILD)/protoc-gen-connect-web: node_modules tsconfig.base.json packages/protoc
 	@mkdir -p $(@D)
 	@touch $(@)
 
-$(BUILD)/connect-core: $(GEN)/connect-core node_modules tsconfig.base.json packages/connect-core/tsconfig.json $(shell find packages/connect-core/src -name '*.ts')
+$(BUILD)/connect-core: $(GEN)/connect-core node_modules tsconfig.base.json packages/connect-core/tsconfig.json $(shell find packages/connect-core/src -name '*.ts') $(shell find packages/connect-core -name 'protocol-*.js')
```

Comment:
I'm a bit worried this is going to break if we ever add a `protocol-*.ts` file in `packages/connect-core/src` - it will be transpiled to `packages/connect-core/dist`, and then `find` will find it.

What do you think about using `packages/connect-core/*.js` as a prerequisite instead?

Reply:
Works for me.  Changed.